Source/FoundationExtensions.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FoundationExtensions.swift
+//  Rex
+//
+//  Created by Ilya Laryionau on 10/05/15.
+//  Copyright (c) 2015 Neil Pankey. All rights reserved.
+//
+
+import ReactiveCococa
+
+extension NSData {
+    /// Read the data at the URL.
+    /// Sends the data or the error.
+    class func rac_dataWithContentsOfURL(url: NSURL, options: NSDataReadingOptions = NSDataReadingOptions.allZeros) -> SignalProducer<NSData, NSError> {
```

Comment:
This should probably use `rex` for a prefix instead of `rac`. I expect some of these may be added to RAC proper in the future and I wouldn't want to collide with them if they aren't removed from here right away.